api/checks/webhook.go

```diff
@@ -254,6 +290,10 @@ func handleCheckRunEvent(
 	return false, nil
 }
 
+// handlePullRequestEvent monitors for pull requests from forks, ensuring that
```

Comment:
The "monitors" wording makes it sounds like this might be something other than an entirely reactive webhook handler.